ipi/engine/thermostats.py

```diff
@@ -504,11 +519,17 @@ def __init__(self, temp=1.0, dt=1.0, tau=1.0, ethermo=0.0, scale=1.0):
               be non-zero if the thermostat is initialised from a checkpoint file.
            scale: A float used to reduce the intensity of the PILE thermostat if
               required.
+           pilect: centroid mode temperature (if different from ensemble and set by input).
+              Default of 0.0 means it is not used and all modes have equal temperatures.
         """
 
         super(ThermoPILE_G, self).__init__(temp, dt, tau, ethermo)
         dself = dd(self)
         dself.pilescale = depend_value(value=scale, name="pilescale")
+        dself.pilect = depend_value(value=pilect, name="pilect")
```

Comment:
is this duplicated, considering pile_g inherits from pile_l ?

Reply:
I think pilescale is also duplicated. It does not hurt, but it is not nice.